py/examples/scripts/ChicagoCrimeDemo.py

```diff
@@ -222,12 +215,12 @@ def score_event(crime, model, censusTable):
     probOfArrest = predictTable["true"][0,0]
     return probOfArrest
 
-for crime in crime_examples:
-    arrestProbGBM = 100*score_event(crime, gbm_model, df_census)
-    arrestProbDLM = 100*score_event(crime, dl_model, df_census)
+for i in crime_examples:
```

Comment:
I changed that because `crime` is defined above this as a function.  This is not an issue if you run the notebook as is. If you want to, for example, create more new `crime_examples` later on, then you will get an error because now `crime` is no longer a function.